e2e/scripts/smoke_test.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+
+test_source_repo=$(pwd)
+docker_test_repo=/workspace/go/src/github.com/rook/rook
+git_smoke_test_directory=github.com/rook/rook/e2e/tests/smoke
+container_image=quay.io/quantum/rook-test
+tmp_docker_sock_path=/tmp/docker.sock
+results_dir=results
+set_go_env_var="export GOPATH=/workspace/go && export GOROOT=/usr/lib/go"
+
+#create the rook infrastructure container
+rook_infra::create() {
+    export id=$(docker run \
+        -itd \
+        --network=host \
+        -e container=docker \
+        --privileged \
+        --security-opt=seccomp:unconfined \
+        --cap-add=SYS_ADMIN \
+        -v /dev:/dev \
+        -v /sys:/sys \
+        -v /sys/fs/cgroup:/sys/fs/cgroup \
+        -v /sbin/modprobe:/sbin/modprobe \
+        -v /lib/modules:/lib/modules:rw \
+        -v /var/run/docker.sock:${tmp_docker_sock_path} \
+        -v ${test_source_repo}:/${docker_test_repo} \
```

Comment:
this extra forward slash isn't needed since `docker_test_repo` already starts with one.